.sentry_dsn.txt

```diff
@@ -0,0 +1 @@
+https://6bc2b763b60e4651ac237871056eb21b@sentry.io/1445091
```

Comment:
probably dont want this?

Reply:
We do. It is the only way to send events.